Dockerfile

```diff
@@ -22,15 +22,20 @@ COPY src/package.json ${GITLAB_CE_PAGES_WEBHOOK_DIR}/
 RUN npm install
 RUN npm install -g nodemon
 
+ENV TINI_VERSION v0.10.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
+
 COPY entrypoint.sh /
 COPY config/nginx.conf /etc/nginx/nginx.conf
 RUN rm -f /etc/nginx/conf.d/*
+RUN chown -R nginx: /var/log/nginx
```

Comment:
According to https://blog.docker.com/2015/04/tips-for-deploying-nginx-official-image-with-docker/ NGINX image will redirect `access.log` to `/dev/stdout`, and redirect `error.log` to `/dev/stderr`. Why is it complaining?
![image](https://cloud.githubusercontent.com/assets/2035734/19418390/7af34d46-93f5-11e6-8140-a3a0d6983f7c.png)